MoltenVK/MoltenVK/Commands/MVKCommandEncoderState.mm

```diff
@@ -156,6 +169,19 @@
 	}
 }
 
+VkRect2D MVKScissorCommandEncoderState::getUnionVkRect() {
+	VkRect2D rslt = {};
+	auto& usingScissors = _scissors.size() > 0 ? _scissors : _dynamicScissors;
+	if ( !usingScissors.empty() ) {
+		rslt = usingScissors[0];
+		size_t sCnt = usingScissors.size();
+		for (uint32_t sIdx = 1; sIdx < sCnt; sIdx++) {
+			rslt = mvkVkRect2DUnion(rslt, usingScissors[sIdx]);
+		}
+	}
+	return rslt;
+}
+
```

Comment:
Is this used for anything?

Reply:
It's a companion to the almost identical `MVKViewportCommandEncoderState::getUnionVkRect()`, but it's not used right now.

TBH...during this, I had experimented with using either (or both), viewport or scissor rect as the mechanism for trimming the render area down to what is actually being rendered, and eventually settled on viewport. 

Scissor seemed like a possible alternative because it could trim it down further if scissor is small. But then I realized scissor doesn't really have an impact on tiles, since the pixel probably has to exist in a tile in order for the pixel to be tested by the scissor test. And on top of that, scissors are often left sized to the entire render area anyway, so it wouldn't help much.

Anyway...I left it in since it is similar to `MVKViewportCommandEncoderState::getUnionVkRect()`, thinking it might be useful at some point. But it can certainly come out if that's never going to be a possibility.